Yehor/reactNetflix/src/redux/actions/moviesActions.ts

```diff
@@ -0,0 +1,62 @@
+import IMovie from "../../interfaces/IMovie";
+
+export enum MoviesTypes {
+  LoadData = "LoadData",
+  ReceivedData = "ReceivedData",
```

Comment:
LoadData not really a meaningful name. What data ? You should specify it in action type